src/icesat2_tracks/analysis_db/B02_make_spectra_gFT.py

```diff
@@ -1,32 +1,40 @@
+import sys
+
+
 """
 This file open a ICEsat2 track applied filters and corections and returns smoothed photon heights on a regular grid in an .nc file.
 This is python 3
 """
 
-import copy
-import datetime
-import h5py
-import time
-import sys
+from icesat2_tracks.config.IceSAT2_startup import (
+    mconfig,
+    plt,
+)
 
+from threadpoolctl import threadpool_info, threadpool_limits
+from pprint import pprint
 import numpy as np
 import xarray as xr
-from pprint import pprint
-from scipy.ndimage.measurements import label
-from threadpoolctl import threadpool_info, threadpool_limits
 
-import icesat2_tracks.ICEsat2_SI_tools.generalized_FT as gFT
+import h5py
 import icesat2_tracks.ICEsat2_SI_tools.io as io
 import icesat2_tracks.ICEsat2_SI_tools.spectral_estimates as spec
-import icesat2_tracks.local_modules.m_general_ph3 as M
-import icesat2_tracks.local_modules.m_spectrum_ph3 as spicke_remover
+
+import time
+import imp
+import copy
+import icesat2_tracks.ICEsat2_SI_tools.spicke_remover as spicke_remover
+import datetime
+import icesat2_tracks.ICEsat2_SI_tools.generalized_FT as gFT
+from scipy.ndimage.measurements import label
```

Comment:
I think this import throws a deprecation warning.
```suggestion
from scipy.ndimage import label
```
Maybe an issue should be created as I've seen this elsewhere in the package.

Reply:
done!